### PR TITLE
Init folder

### LINKED ISSUE #39: Init folder
image

### PR DESCRIPTION
**Pull request by CBinh710**
## Design:
image
### Description:
- Create assets folder.
- Add fonts and images to assets folder.
- Create styles folder base on 7-1 pattern

### CODE REVIEW

File: practice-1/src/styles/main.css [hunk 1]
```diff
@@ -0,0 +1,5 @@
+@import url(abstracts/index.css);
+@import url(base/index.css);
+@import url(layout/index.css);
```
Comment (hungle-agilityio) [inline practice-1/src/styles/main.css]: @CBinh710 please double check, you should be import correct order.

Comment (CBinh710) [reply practice-1/src/styles/main.css]: I fixed all

Comment (CBinh710) [reply practice-1/src/styles/main.css]: thanks for your comment

File: practice-1/src/styles/base/reset.css [hunk 2]
```diff
@@ -0,0 +1,49 @@
+/* Reset CSS */
```
Comment (phuocduong-agilityio) [inline practice-1/src/styles/base/reset.css:1]: Format

File: practice-1/src/styles/abstracts/variables.css [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  --color-dark: #000000;
+  --color-light: #06CAF5;
+  --color-filter-violet: #7D06F5;
+  --color-filter-blue: #063AF5;
+  --color-filter-pink: #F506A4;
+  --color-filter-orange: #F57906;
+  --color-filter-green: #00C12B;
+
+  --font-family-base: satoshi-regular;
+  --font-family-secondary: inter;
```
Comment (phuocduong-agilityio) [inline practice-1/src/styles/abstracts/variables.css]: Have you try this work?

File: practice-1/src/styles/main.css (hunk 1 above)
Comment (phuocduong-agilityio) [reply practice-1/src/styles/main.css]: @CBinh710 Wrong order. It's should be from common to specific one. It's mean layout.css can not be here

Comment (CBinh710) [reply practice-1/src/styles/main.css]: I changed order, thanks for your comment

File: practice-1/src/styles/base/reset.css (hunk 2 above)
Comment (CBinh710) [reply practice-1/src/styles/base/reset.css:1]: I changed indent to 2, thanks

File: practice-1/src/styles/abstracts/variables.css (hunk 3 above)
Comment (CBinh710) [reply practice-1/src/styles/abstracts/variables.css]: I fixed it, Thanks for your contribution

File: practice-1/src/styles/base/reset.css (hunk 2 above)
Comment (phuocduong-agilityio) [reply practice-1/src/styles/base/reset.css:1]: @CBinh710 Not indent only, for each style element, should split new line:

Ex:

p,
i,
b,

File: practice-1/src/index.html [hunk 4]
```diff
@@ -4,14 +4,9 @@
...
-    <!-- <script src="./exercises/object.js"></script> -->
-    <!-- <script src="./exercises/function.js"></script> -->
-    <!-- <script src="./exercises/class.js"></script> -->
-     <!-- <script src="./exercises/async.js"></script> -->
-     <script src="./exercises/promise.js"></script>
+    <link rel="stylesheet" href="./styles/main.css">
   </head>
   <body>
-    <p>Hello world</p>
+    <p style="font-family: satoshi-bold;">Hello world</p>
```
Comment (phuocduong-agilityio) [inline practice-1/src/index.html]: Remove this

Comment (CBinh710) [reply practice-1/src/index.html]: I removed it, thanks

**Pull request merged**